### DISCUSSION TITLE
"Double click group to set title" setting

### DISCUSSION THREAD

**Discussion by Gerkinfeltser** [mape/ComfyUI-mape-Helpers#16] [Ideas]
The `Double click node to set title` is wonderful (among many other improvements).
Would it be possible to do the same for groups?
Thank you!

**Comment by mape** [comment]
It should work if you use `Shift + Double Click`

https://github.com/mape/ComfyUI-mape-Helpers/assets/25655/e8ea40fe-1d6e-4cce-a2b6-6ee79fdf176d

**Reply by Gerkinfeltser** [reply]
Thanks for the video & explaination. It appears this functionality doesn't work in Chrome though. I'll try another browser.

**Reply by heianzhexx** [reply]
Same problem +1
